### PR TITLE
added labs and unit tests for accessibility

### PR DESCRIPTION
**Pull request by SurgeArrester**

### CODE REVIEW

File: problems/accessibility/__init__.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+@check50.check(student_constructor)
+def student_not_update_lower():
+    """getGrades() with assignment number out of bounds above"""
+    check50_java.junit5.run_and_interpret_test(
+        classpaths=['tests/'],
+        args=['--select-method', 'StudentTest#testOutOfBoundsUpper'])
+
+
+@check50.check(student_constructor)
+def student_not_update_lower():
```
Comment (patricktotzke) [inline problems/accessibility/__init__.py:132]: this needs changing

**Review by patricktotzke — CHANGES_REQUESTED** [review]

File: problems/accessibility/__init__.py (hunk 1 above)
Comment (SurgeArrester) [reply problems/accessibility/__init__.py:132]: no it doesn't!

File: labs/accessibility/Student.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    ? getHasSubmitted() {
+        return hasSubmitted;
+    }
+
+    ? getGrades() {
+        ?
+    }
+
+    ? getTotalGrade() {
```
Comment (patricktotzke) [inline labs/accessibility/Student.java:32]: perhaps comment out the whole thing and ask students to write uncomment and finish it?
The check50_java check should survive a missing method and simply fail that check if they did not do this.

**Review by patricktotzke — CHANGES_REQUESTED** [review]

**Comment by pazz** [comment]
closed in favour of #4

**Pull request closed without merging**